mongodol/tests/trans_test.py

```diff
@@ -2,8 +2,9 @@
 
 
 def test_set_key_and_data_fields():
-    from dol.base import MongoCollectionCollection, MongoCollectionFirstDocPersister
-    from dol.trans import set_key_and_data_fields
+    from mongodol.base import MongoCollectionCollection
+    from mongodol.stores import MongoCollectionFirstDocPersister
+    from mongodol.trans import set_key_and_data_fields
```

Comment:
Should this really be specific to mongo? I think this string/tuple/dict transformations are quite common and in fact there are tools for it (some being refactored right now by me) in dol already.

Reply:
I think we can move it to dol. But it is not there yet, so I just used the one in mongodol